MMM-COVID19.js

```diff
@@ -79,24 +80,36 @@ Module.register("MMM-COVID19", {
     // header row
     var headerRow = document.createElement("tr"),
         headerconfirmedCell = document.createElement("td"),
+        headerNewConfirmedCell = document.createElement("td"),
         headerCountryNameCell = document.createElement("td"),
         headerrecoveredCell = document.createElement("td"),
         headerdeathsCell = document.createElement("td"),
+        headerNewDeathsCell = document.createElement("td"),
         headeractiveCell = document.createElement("td")
 
     headerCountryNameCell.innerHTML = ''
     headerconfirmedCell.className = 'number confirmed ' + this.config.headerRowClass
     headerconfirmedCell.innerHTML = 'Confirmed'
+    headerNewConfirmedCell.className = 'number confirmed ' + this.config.headerRowClass
+    headerNewConfirmedCell.innerHTML = 'New Cases'
     headerdeathsCell.className = 'number deaths ' + this.config.headerRowClass
     headerdeathsCell.innerHTML = 'Deaths'
+    headerNewDeathsCell.className = 'number deaths ' + this.config.headerRowClass
+    headerNewDeathsCell.innerHTML = 'New Deaths'
     headerrecoveredCell.className = 'number recovered ' + this.config.headerRowClass
     headerrecoveredCell.innerHTML = 'Recovered'
     headeractiveCell.className = 'number active ' + this.config.headerRowClass
     headeractiveCell.innerHTML = 'Active'
 
     headerRow.appendChild(headerCountryNameCell)
     headerRow.appendChild(headerconfirmedCell)
+    if (this.config.delta) {
+      headerRow.appendChild(headerNewConfirmedCell)
+    }
     headerRow.appendChild(headerdeathsCell)
+    if (this.config.delta) {
+      headerRow.appendChild(headerNewDeathsCell)
+    }
```

Comment:
I would change something here.
inside the if block, I would put all related info
```
   if (this.config.delta) {
      headerRow.appendChild(headerNewConfirmedCell)
      headerRow.appendChild(headerNewDeathsCell)
    }

Reply:
the problem is that the order in which the children are appended matters.
IMHO the alteration you suggested would not result in better code:

~~~
if (this.config.delta) {
      headerRow.appendChild(headerNewConfirmedCell)
      headerRow.appendChild(headerdeathsCell)
      headerRow.appendChild(headerNewDeathsCell)
} else {
      headerRow.appendChild(headerdeathsCell)
}
~~~

In the end I settled for this code because IMHO integrating more columns would become more straight forward as the alternative would lead to awkward nesting of if blocks.

Reply:
ok, good